.github/workflows/node.yml

```diff
@@ -161,19 +173,27 @@ jobs:
               run: |
                   yum install -y nodejs
 
+            - name: Install shared software dependencies
+              uses: ./.github/workflows/install-shared-dependencies
+              with:
+                  os: "amazon-linux"
+                  target: "x86_64-unknown-linux-gnu"
+                  github-token: ${{ secrets.GITHUB_TOKEN }}
+
+            - name: Install redis
+              uses: ./.github/workflows/install-redis
+              with:
+                  redis-version: 6.2.14
+
             - name: Build Node wrapper
               uses: ./.github/workflows/build-node-wrapper
               with:
                   os: "amazon-linux"
                   target: "x86_64-unknown-linux-gnu"
                   github-token: ${{ secrets.GITHUB_TOKEN }}
 
-            - name: Create a symbolic Link for redis6 binaries
-              working-directory: ./node
-              run: |
-                  ln -s /usr/bin/redis6-server /usr/bin/redis-server
-                  ln -s /usr/bin/redis6-cli /usr/bin/redis-cli
-
             - name: Test compatibility
-              run: npm test -- -t "set and get flow works"
               working-directory: ./node
+              run: |
+                npm install --save-dev jest
```

Comment:
This change seems unrelated to the Redis installation. What is this for?

Reply:
I found CI failed because of missing `jest`. I have no idea how it works on upstream, unfortunately.

Reply:
Maybe worth trying without `jest` on upstream in a draft and adding it in the real PR if still needed. Anyway, not a blocker.

Reply:
Sure, thanks